sfm_pc/templatetags/language_url.py

```diff
@@ -0,0 +1,26 @@
+import json
+
+from  more_itertools import unique_everseen
+
+from django import template
+from django.conf import settings
+from django.template.defaultfilters import stringfilter
+
+register = template.Library()
+
+@register.filter
+def create_select2_data(select_lang):
+  data_array = ["English", "French", "Spanish"]
```

Comment:
This is a style suggestion more than anything else, but I might instantiate `trans_obj` first (or use a global settings variable, see my comment below) and then create `data_array` like so:

```python
data_array = trans_obj.values()
```

to make the relationship between the two objects explicit.

Reply:
Very nice. Ended up with this (thanks, Python 3):

`data_array = list(trans_obj.values())`

Reply:
Love Python3!